packages/ui/src/components/ChatMessage/ChatContent.tsx

```diff
@@ -0,0 +1,278 @@
+import { utils } from '@tloncorp/shared';
+import {
+  Block,
+  Story,
+  VerseBlock,
+  VerseInline,
+  isImage,
+} from '@tloncorp/shared/dist/urbit/channel';
+import {
+  Inline,
+  isBlockCode,
+  isBlockquote,
+  isBold,
+  isBreak,
+  isInlineCode,
+  isItalics,
+  isLink,
+  isShip,
+  isStrikethrough,
+} from '@tloncorp/shared/dist/urbit/content';
+import { findLastIndex } from 'lodash';
+import { Image, SizableText, View, YStack } from 'tamagui';
+
+import { Button } from '../Button';
+import ShipName from '../ShipName';
+
+function ShipMention({ ship }: { ship: string }) {
+  return (
+    <Button
+      // TODO: implement this once we have a profile screen or sheet
+      // onPress={() => naivigate('Profile', { ship })}
+      backgroundColor="$blueSoft"
```

Comment:
This is prob placeholder, so feel free to ignore, but with this many style props for something like this, I'd start thinking about adding a variant.